app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,15 @@
+<h1>Dish: <%= @recipe.name %></h1>
+<h1>Complexity: <%= @recipe.complexity %></h1>
+<h1>Genre: <%= @recipe.genre %></h1>
+<h2>INGREDIENTS:</h2>
+<h3><% @ingredients.each do |ingredient| %></h3>
+  <ul>
+    <li><%= ingredient.name %></li>
+  </ul>
+<% end %>
+<%= form_with url: "/recipes/#{@recipe.id}", method: :post, local: true do |form| %>
+  <%= form.label :New %>
```

Comment:
`New` is not the most descriptive field name. Consider `ingredient_id` or something more representative of what the data refers to